RecoLocalCalo/EcalRecProducers/python/ecalRecHit_cfi.py

```diff
@@ -72,12 +72,15 @@
     #  ~64 GeV is the TP saturation level
     logWarningEtThreshold_EB_FE = cms.double(50),# in EB logWarningThreshold is actually in E (GeV)
     logWarningEtThreshold_EE_FE = cms.double(50),# in EE the energy should correspond to Et (GeV) but the recovered values of energies are not tested if make sense
+    singleChannelRecoveryMethod = cms.string("BDTG"), #default NeuralNetworks
+    singleChannelRecoveryThreshold = cms.double(0.70),##default 8
+    sum8ChannelRecoveryThreshold = cms.double(0.),##default 8
+    bdtWeightFileNoCracks = cms.FileInPath("RecoLocalCalo/EcalDeadChannelRecoveryAlgos/data/TMVARegression_trainingWithAliveCrystalsAllRH_8GT700MeV_RelEne_noCracks_rightTAG_data_BDTG.weights.xml"),
+    bdtWeightFileCracks = cms.FileInPath("RecoLocalCalo/EcalDeadChannelRecoveryAlgos/data/TMVARegression_trainingWithAliveCrystalsAllRH_8GT700MeV_RelEne_onlyCracks_rightTAG_data_BDTG.weights.xml"),
```

Comment:
IIUC, these files are not available yet in a cms-data repo pull request.
Please also check if the name can be shortened.
It looks like ```noCracks.xml``` and ```onlyCracks.xml``` perhaps with some "v1" for future version tracking is enough.